AIDriver.lua

```diff
@@ -216,6 +216,10 @@ end
 
 --- Dismiss the driver
 function AIDriver:dismiss()
+	if self.collisionDetector then		-- restore the default direction of the colli boxes
+		self.collisionDetector:deleteTriggers()
```

Comment:
If we want to do it this way then we need a TrafficCollision:reset() or whatever function to hide its internals.

Reply:
In AIDriver the internals of the collisionDetector are already accessed, so I did the same.
Should something be changed here for now?

Reply:
Yes please, add a new function like reset()  to wrap the delete/create

Reply:
done with CollisionDetector:reset()